tests/unit/models/test_input.py

```diff
@@ -211,7 +211,7 @@ def test_invalid_package_paths(self, path: str, expect_error: str, tmp_path: Pat
             )
```

Comment:
**Commit message:**

Hold on, isn't the URL actually pointing to a private resource? IOW Isn't the board private to members only?

Reply:
You mean "issues.redhat.com"? If yes, why?

Reply:
Well, I normally just add the Jira ticket, not the full URL. But I don't think we have a strict rule about this.

Reply:
> You mean "issues.redhat.com"? If yes, why?

Because unlike 'STONEBLD-1750' which is just an opaque string literal telling absolutely nothing to an outsider contributor (which is also a good reason not to have it in a public commit message at all, minus synchrozation back to the Jira instance which is the only reason we stick to using it), a URL to a private resource in a public commit message is IMO an antipattern because everyone can (and they will) click on the URL hoping they could get more information about a given commit (if the message itself isn't descriptive enough). Why do I even mention this as a problem? Let's mention bugzilla as an example. Before Red Hat moved to Jira for tracking (although bugs are to be tracked publicly) people used to put bugzilla links into commit messages as an additional source of information because bugzilla clearly stated the overall problem a given patch is trying to address along with information about reproducers, which products were impacted in what versions, etc. but ONLY when the link pointed to a public bugzilla, because links to private bugzillas would be pointless to outside contributors. So non-RH contributors are actually very much used to put and find this kind of information in commit messages and if we continue putting private URLs in commit messages, that is what creates the anti-pattern.